cetloxblock/setup.py

```diff
@@ -0,0 +1,41 @@
+"""Setup for cetloxblock XBlock."""
+
+import os
+
+from setuptools import setup
+
+
+def package_data(pkg, roots):
+    """Generic function to find package_data.
+
+    All of the files under each of the `roots` will be declared as package
+    data for package `pkg`.
+
+    """
+    data = []
+    for root in roots:
+        for dirname, _, files in os.walk(os.path.join(pkg, root)):
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
+
+    return {pkg: data}
+
+
+setup(
+    name='cetloxblock-xblock',
+    version='1.0.0',
+    description='cetloxblock XBlock',   # TODO: write a better description.

```

Comment:
You can probably just use a re-worded version of the `Goal` section in your `README.md`:

"An XBlock that embeds the CET Player platform into Open edX. The CET player contains the ability to create personlized learning path for each student."

Reply:
In addition, you can add a `long_description` which contains the entire `README.md`. Example: https://github.com/edx/edx-enterprise/blob/master/setup.py#L43 -- notice how they grab the README file pretty easily and just stick it there.

Reply:
Implemented.
For this I removed the upper level directory, to have the readme file in the same level as the setup.py. This corresponds to your [1st (closed) pull request](https://github.com/ILCET/lo-activity-xblock/pull/1).

But where can I see the long_description in action?

Reply:
The information in setup.py forms the package information provided if you package and distribute your code.  This is optional, since we can install directly from github, but some packages choose to do this.

For the edx-enterprise example that @UmanShahzad  gave, they distribute that package to https://pypi.org/project/edx-enterprise/, and so the long description becomes the package information shown there.

More info: http://python-packaging.readthedocs.io/en/latest/metadata.html